docs/tutorials/developer-journey/level-5/5.5-SNS-tutorial.md

```diff
@@ -0,0 +1,1097 @@
+
+SNSs are provided as a system functionality. For an SNS to be launched, the NNS community must vote on and approve the SNS's original canister code. Once the SNS is launched, the code for the SNS's canisters is maintained by the IC and is changed through SNS proposals that are voted on by the community. All approved SNS canister versions are stored on an NNS canister known as the SNS Wasm Modules canister (SNS-W). Recall that all canisters are compiled into Wasm before being deployed on the IC.
+
+When an SNS is created, the SNS-W canister is responsible for deploying the latest version of the SNS's canisters. When an SNS is updated, an NNS proposal is used to add a new version of the SNS's canisters to the SNS-W canister. Then, the SNS's community decides via SNS proposal if they'd like to adopt the new, approved version into the SNS. Once approved via proposal, the updates are applied to the SNS's canisters. 
+
+All SNSs are hosted on the SNS subnet, which is exclusively used to host SNSs. This allows for simplified verification for end users, since end users can verify that an SNS is running on the SNS subnet, and therefore trust that the SNS's underlying code has been approved by the NNS community. 
+
+An SNS is comprised of the following canisters:
+
+- **SNS governance canister**: The governance canister defines who can participate in the SNS's governance decisions and automatically triggers the execution of all decisions. This canister stores the SNS proposals and a record of the neurons that define who the governance participants are. When a proposal is adopted, the governance system automatically triggers the execution of the proposal by calling a defined method. In most instances, the decision of the proposal is executed fully on-chain. 
+
+- **SNS ledger and archive canisters**: The ledger canister implements the ICRC-1 token standard to create a unique token for each SNS. To learn more about ICRC-1 tokens, refer to the previous module [4.2: ICRC-1 tokens](../level-4/4.2-icrc-tokens.md) where you deployed an ICRC-1 ledger then created an ICRC-1 token. This canister stores the SNS's ledger, which keeps track of how many accounts own SNS tokens, the amount owned by each account, and the history of transactions for the SNS's token. Since the canister has limited memory, to keep the ledger's full history, the ledger canister spawns archive canisters to store the ledger's block history. 
+
+- **SNS index canister**: The index canister keeps a record of which transactions are relevant for a given account. This provides the ability for wallets and other frontends to show all transactions for an account without requiring every frontend to implement this functionality themselves. When the NNS creates a new SNS, it will deploy the SNS's index canister as part of the deployment process. To deploy the index canister locally, follow the instructions found [here](/docs/current/developer-docs/integrations/icrc-1/icrc1-index-setup). 
```

Comment:
```suggestion
- **SNS index canister**: The index canister keeps a record of which transactions are relevant for a given account on the SNS ledger. This provides the ability for wallets and other frontends to show all transactions for an account without requiring every frontend to implement this functionality themselves.
```

I think the part wrt deployment holds for all these canisters, so I am not sure we have to make this explicit only for the index canister.